.github/workflows/ci_tests.yml

```diff
@@ -34,6 +34,8 @@ jobs:
           pip install pytest
           pip install -e .[test]
           if [ -f requirements.txt ]; then pip install -r requirements.txt; fi
+      - name: Test with mypy
```

Comment:
Would it be worth using the mypy action?
https://github.com/marketplace/actions/mypy-action

Reply:
I've never noticed a difference. I think the mypy action just installs it for you. But it should already have been pip installed in the test group.

Reply:
Resolved